main/main.go

```diff
@@ -25,13 +25,20 @@ import (
 	"github.com/ava-labs/awm-relayer/vms/vmtypes"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/spf13/pflag"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 )
 
+var version = "v0.0.0-dev"
+
 func main() {
 	fs := config.BuildFlagSet()
+	if err := fs.Parse(os.Args[1:]); err != nil {
```

Comment:
Rather than calling `os.Args` twice, let's define `args := os.Args[1:]` as the first line of this function and reuse it.

Reply:
Nice catch ! I figured out `config.BuildViper` does not need anymore the os.Args, so it will only be used once after my changes